Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListItem.swift

```diff
@@ -129,24 +129,45 @@ public struct ChatChannelListItem: View {
 
 /// View for the avatar used in channels (includes online indicator overlay).
 public struct ChannelAvatarView: View {
-
-    var avatar: UIImage
-    var showOnlineIndicator: Bool
-    var size: CGSize = .defaultAvatarSize
-
```

Comment:
Slightly messy because this is a public type and I needed to pass in `ChatChannel` here. Should be cleaned up in v5.